HW3/Task1.py

```diff
@@ -0,0 +1,12 @@
+while True:
+    test_string = str(input('Write here your string:'))
```

Comment:
`str` is redundant here, input already returns string value
```suggestion
    test_string = input('Write here your string:')
```